test/e2e/config/bmcs-fixture.yaml

```diff
@@ -5,7 +5,15 @@
   password: password
   address: "redfish+http://192.168.222.1:8000/redfish/v1/Systems/bmo-e2e-0"
   bootMacAddress: "00:60:2f:31:81:01"
+  networks:
+  - name: "baremetal-e2e"
+    macAddress: "00:60:2f:31:81:01"
+    ipAddress: "192.168.222.122"
```

Comment:
Can we leave it out for fixture? Then we keep the file smaller and people don't get confused thinking that the fixture tests can actually use this. Plus, we test that the configuration is optional.